apps/mobile/src/index.tsx

```diff
@@ -26,6 +22,11 @@ function App() {
   })
 
   const scheme = useColorScheme()
+  const theme = scheme === 'dark' ? DarkTheme : DefaultTheme
+  useEffect(() => {
+    SystemUI.setBackgroundColorAsync(theme.colors.card)
```

Comment:
oh tricky, I guess this is helpful when the screen rotates and you see the background temporarily?

Reply:
Yeah, I didn't think about it, but you are right - that's another case. In our case, popping modal view was showing white background!